### DISCUSSION TITLE
How can add array inside another array? using useFieldArray()?

### DISCUSSION THREAD

**Discussion by PetrusBorges** [react-hook-form/react-hook-form#11875] [General]
basically i have a array of houses e inside of each a need do push a array of rooms, how can i be able to create any rooms i need for eatch houses i created in form? 

this exemple that i show dont works for houses.${index}.rooms

fieldNote for houses works well, but fieldRooms to name: `houses.${houseIndex}.rooms` dont works

![image](https://github.com/react-hook-form/react-hook-form/assets/105453766/34631877-1d28-4056-a1cc-c739e964d233)

**Comment by selrond** [comment]
from the [docs](https://react-hook-form.com/docs/usefieldarray): 
> Name of the field array. **Note**: Do not support dynamic name.

you can’t nest `useFieldArray` like this, if you want nested arrays of fields, you have to give them dynamic name when registering / using `<Controller />` like this:

`houses.${houseIndex}.rooms.${roomIndex}.name` etc
